src/mlpack/methods/ann/layer/transposed_convolution.hpp

```diff
@@ -163,6 +163,45 @@ class TransposedConvolution
   //! Modify the output height.
   size_t& OutputHeight() { return outputHeight; }
 
+  //! Get the input size
+  size_t const& InputSize() const { return inSize; }
+
+  //! Get the output size.
+  size_t const& OutputSize() const { return outSize; }
+
+  //! Get the kernel dimensions.
+  std::tuple<size_t, size_t> const& KernelDims() const
```

Comment:
I was wondering if it make sense to use the same naming schema as Keras does, so ``KernelSize`` and ``strides``. What do you think?

Reply:
```KernelSize``` (, ```strides```, ```padding```) clearly sounds better, but what we currently have is a consistent format where the required parameter is followed by **Dims** (but again we already have a few functions like ```OutputSize()```). Another consideration might be that KernelSize, strides, etc. are names of the respective arguments, while for obtaining the kernel size from a Keras model one would probably have to call ```layer.output_shape```. However, given a choice I would probably prefer what you have suggested, so I'll go ahead and change the function names. Thanks for the comment.